share/completions/loadkeys.fish

```diff
@@ -0,0 +1,15 @@
+complete -fc loadkeys -a "(find /usr/share/kbd/keymaps/ -type f | string replace -rf '.*/(.*).map.gz' '\$1')"
+
+complete -c loadkeys -s C -l console -d "Console device to use" -x -a "(command ls /dev/tty*)"
```

Comment:
Use `path filter --type=char /dev/tty*` instead of `command ls` (which might have portability issues, will print directories, requires starting another command, and will print an error if no files match).